network/lwip2transport/doc.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+The network/lwip2transport package uses the lwIP (A Lightweight TCP/IP stack) library to translate between IP packets
```

Comment:
Perhaps link to lwip site? We should also mention that we use lwip via github.com/eycorsican/go-tun2socks